tests/conftest.py

```diff
@@ -11,60 +11,57 @@
 
 
 @pytest.fixture(autouse=True)
-def test_env():
-    os.environ = {
-        "AWS_ACCESS_KEY_ID": "testing",
-        "AWS_DEFAULT_REGION": "us-east-1",
-        "AWS_SECRET_ACCESS_KEY": "testing",
-        "AWS_SECURITY_TOKEN": "testing",
-        "AWS_SESSION_TOKEN": "testing",
-        "ALMA_CHALLENGE_SECRET": "itsasecret",
-        "ALMA_POD_EXPORT_JOB_NAME": "PPOD Export",
-        "ALMA_TIMDEX_EXPORT_JOB_NAME_PREFIX": "TIMDEX Export",
-        "ALMA_BURSAR_EXPORT_JOB_NAME_PREFIX": "Export to bursar",
-        "LAMBDA_FUNCTION_URL": "http://example.com/lambda",
-        "PPOD_STATE_MACHINE_ARN": "arn:aws:states:us-east-1:account:stateMachine:"
-        "ppod-test",
-        "TIMDEX_STATE_MACHINE_ARN": "arn:aws:states:us-east-1:account:stateMachine:"
-        "timdex-test",
-        "BURSAR_STATE_MACHINE_ARN": "arn:aws:states:us-east-1:account:stateMachine:"
-        "bursar-test",
-        "VALID_POD_EXPORT_DATE": "2022-05-23",
-        "WORKSPACE": "test",
-    }
-    return
-
-
-@pytest.fixture()
+def _test_env():
+    os.environ["AWS_ACCESS_KEY_ID"] = "testing"
+    os.environ["AWS_DEFAULT_REGION"] = "us-east-1"
+    os.environ["AWS_SECRET_ACCESS_KEY"] = "testing"
+    os.environ["AWS_SECURITY_TOKEN"] = "testing"
+    os.environ["AWS_SESSION_TOKEN"] = "testing"
+    os.environ["ALMA_CHALLENGE_SECRET"] = "itsasecret"
+    os.environ["ALMA_POD_EXPORT_JOB_NAME"] = "PPOD Export"
+    os.environ["ALMA_TIMDEX_EXPORT_JOB_NAME_PREFIX"] = "TIMDEX Export"
+    os.environ["ALMA_BURSAR_EXPORT_JOB_NAME_PREFIX"] = "Export to bursar"
+    os.environ["LAMBDA_FUNCTION_URL"] = "http://example.com/lambda"
+    os.environ[
+        "PPOD_STATE_MACHINE_ARN"
+    ] = "arn:aws:states:us-east-1:account:stateMachine:ppod-test"
+    os.environ[
+        "TIMDEX_STATE_MACHINE_ARN"
+    ] = "arn:aws:states:us-east-1:account:stateMachine:timdex-test"
+    os.environ[
+        "BURSAR_STATE_MACHINE_ARN"
+    ] = "arn:aws:states:us-east-1:account:stateMachine:bursar-test"
+    os.environ["VALID_POD_EXPORT_DATE"] = "2022-05-23"
+    os.environ["WORKSPACE"] = "test"
```

Comment:
Assuming this change was from `ruff` and I think it's okay as-is, but we might want to keep our eyes open for ways in which the testing environment is set across projects.

For one, the repeating `os.environ[KEY] = VALUE` could be performed in a loop from a list of key/value pairs.  And _that_ begs the question whether or not those key/value pairs should be defined in the fixture itself, or somewhere else (example you can define a [`pytest.ini`](https://docs.pytest.org/en/7.1.x/reference/customize.html#pytest-ini) file)